classes/components/forms/publication/PublishForm.inc.php

```diff
@@ -79,5 +82,105 @@ public function __construct($action, $publication, $submissionContext, $requirem
                 'description' => $msg,
                 'groupId' => 'default',
             ]));
+
+        if ($submissionContext->areDoisEnabled()) {
+            $this->addField(new FieldHTML('doi', [
+                'description' => $this->_getDoiMessage(),
+                'groupId' => 'default'
+            ]));
+        }
+    }
+
+    private function _getDoiMessage(): string
```

Comment:
Since we never showed this in OMP before, we don't need to add it in now. This will save us a little bit of a UX headache when DOIs are created on publication, and this UI can't yet show a DOI.

Reply:
If you want to keep this in that's fine too...

Reply:
I'm also happy to get rid of it.